### ISSUE TITLE
NVDA not announcing role of alert in Chrome

### ISSUE THREAD

**Issue by Gcamara14** [nvaccess/nvda#13418]
### Steps to reproduce:

1. Activate Chrome Browser
2. Turn on NVDA
3. Visit/Test - [Alert Example w3](https://www.w3.org/TR/wai-aria-practices-1.2/examples/alert/alert.html)
4.  Visit/Test - [ARIA19: Using ARIA role alert](https://www.w3.org/TR/WCAG20-TECHS/ARIA19.html)

### Actual behavior:

NVDA is not announcing the `alert ` role in Chrome. NVDA does work in firefox.

### Expected behavior:

NVDA should announce the `alert` information when it appears on the screen while using CHROME browser.

#### NVDA installed/portable/running from source:

YES

#### NVDA version:

NVDA 2021.3.3

#### Windows version:

Windows 10 enterprise

#### Name and version of other software in use when reproducing the issue:

Browser: Chrome Version 98.0.4758.102

#### Other information about your system:

N/A

#### Does the issue still occur after restarting your computer?

Yes, and tested with others.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

No, I have not.

#### If NVDA add-ons are disabled, is your problem still occurring?

I do not have add-ons

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

Yes

**Comment by seanbudd** [comment]
Is this a duplicate of #11990?

I have confirmed this issue on Chrome. Notably, "hello" fails to be announced the first time, but is announced correctly when triggering the alert subsequently.

**Comment by Adriani90** [comment]
This seems fixed in Chrome Canary 128 with NVDA 2024.3 Beta. Closing as works for me. If you are still having this issue, please comment and we can reopen.
